src/main/java/com/google/graphgeckos/dashboard/storage/DataRepository.java

```diff
@@ -24,33 +24,32 @@
    * If there already is an entry with the same commit hash, ignores the request.
    *
    * @param entryData a ParsedGitData instance, must have a non-null "commitHash" field
-   * @throws IllegalArgumentException if entryData is null, or {@link 
-   *       com.google.graphgeckos.dashboard.storage.ParsedGitData#validCreateData()
-   *       entryData.validCreateData()} returns false
+   * @return true if operation completed successfully.
```

Comment:
Add _only_ to be more specific
```suggestion
   * @return true only if operation completed successfully.
```

Reply:
Will add this to all `@return` annotations in a separate commit